OpenSim/Simulation/Model/PrescribedForce.h

```diff
@@ -94,11 +89,19 @@ OpenSim_DECLARE_CONCRETE_OBJECT(PrescribedForce, Force);
     /** These are three functions providing the x,y,z measure numbers of the
     torque vector being applied to the body. The coordinate frame in which
     this vector is interpreted depends on the "torqueIsGlobal" property. **/
+
     // Would have been better for this to be a list property. 
     OpenSim_DECLARE_PROPERTY(torqueFunctions, FunctionSet,
```

Comment:
I agree with this comment. If you make it a sized list property we can enforce exactly 3 components.